src/energy_distribution.F90

```diff
@@ -1,7 +1,9 @@
 module energy_distribution
 
-  use constants,     only: ZERO, ONE, TWO, PI, HISTOGRAM, LINEAR_LINEAR
+  use constants,     only: ZERO, ONE, HALF, TWO, PI, HISTOGRAM, LINEAR_LINEAR
   use endf_header,   only: Tabulated1D
+  use hdf5_interface
+  use hdf5
```

Comment:
I think we should keep the 3rd-party imports (i.e. hdf5) separate from the imports of our own modules, like we do with our Python headers.